packages/core/src/resources/Issues.ts

```diff
@@ -24,6 +24,15 @@ export interface TimeStatsSchema extends Record<string, unknown> {
   human_total_time_spent: string | null;
 }
 
+export interface IssueLabelDetailsSchema extends Record<string, unknown> {
+  id: number;
+  name: string;
+  description: null | string;
+  description_html: string;
+  text_color: string;
+  color: string;
+}
+
```

Comment:
Issues, MergeRequest and Epics can all have LabelDetails attached to them.

This interface could probably be moved to `templates/ResourceLabel.ts` and renamed `LabelDetailsSchema`.